scripts/create_broker_secret.py

```diff
@@ -1,8 +1,28 @@
 #! /usr/bin/env python
+
 import sys
 import yaml
 import subprocess
 
+try:
+    import yaml
```

Comment:
You have `import yaml`, so I don't believe you would ever reach `print("No yaml parsing modules installed, try: pip install pyyaml")` as it stands now.

Reply:
It isn't immediately obvious why we are try/catching this. It would be helpful, to me at least, if you could explain why you are doing this.

Reply:
I'm just try-catching it because this isn't a python project and we can't really specifiy python dependencies, so I wanted a nice error message if they don't have the right dependencies installed. I messed up with the above import yaml though, oops.